components/service-operator/apis/storage/v1beta1/s3_types_test.go

```diff
@@ -99,6 +101,14 @@ var _ = Describe("S3Bucket", func() {
 		Expect(o2).ToNot(BeZero())
 	})
 
+	It("should call the AWS API when emptied", func() {
+		client = &sdkfakes.FakeClient{}
```

Comment:
this looks incomplete?

client is not defined, and none of the methods have been configured so I would expect this to panic ...right? :thinking: